tensorflow/contrib/cmake/README.md

```diff
@@ -13,23 +13,59 @@ Linux.
 Current Status
 --------------
 
+TensorFlow release 0.12 officially supports Windows, with a native TensorFlow package 
+provided for Windows 7, 10 and Server 2016.
+The supported languages include Python 3.5 64-bit (from Python.org or Anaconda) and C++. 
+This release enables acceleration with CUDA 8.0 and cuDNN 5.1 compatible GPUs.
+A [pip package](https://pypi.python.org/pypi/tensorflow) was published and the
+installation can be done with a single command:
+
+`C:\> pip install tensorflow`
+
+And for GPU support:
+
+`C:\> pip install tensorflow-gpu`
```

Comment:
I'd remove the content here (lines 16--27) and replace it with something like the following:

CMake can be used to build TensorFlow on Windows. See the [getting started documentation](https://www.tensorflow.org/get_started/os_setup.html#pip-installation-on-windows) for instructions on how to install a pre-built TensorFlow package on Windows.